taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css

```diff
@@ -242,19 +221,15 @@ Styleguide Objects.Section
 /* FP-1462: Extract to new banner component(s) */
 
 /* HACK: To hide vertical overflow by placing sibling sections atop */
+[class*="o-section--style"] { position: relative; }
 /* NOTE: This selector assumes only a banner section has vertical overflow */
 /* CAVEAT: Any banner pop-out el's can NOT overflow atop sibling sections */
 [class*="o-section--style"].o-section--banner { z-index: 0; }
 /* NOTE: A `z-index` > 0 is only necessary for sections before banner */
 /* CAVEAT: Any section pop-out el's can NOT overflow atop section after it */
 [class*="o-section--style"] { z-index: 1; }
 
-
-
-/* Tricks: /* Modifiers: Style */
-/* Tricks: /* Children: Banner Image */
-
-/* HACK: To hide `.o-section--style-…` & `.o-section__banner-image` overflow */
+/* HACK: To hide `.o-section__banner-image` overflow */
```

Comment:
**Explanation.** The `overflow-x: hidden` (below) is still a hack/trick, but the extended background does not benefit from it now (only the banner image does).